HW_module_4_task2.py

```diff
@@ -2,27 +2,28 @@
 import random
 
 # create a list of random number of dicts (from 2 to 10)
-def create_list_of_dict(a, b, c=4, e=5, f=1):
-    dict_List = []
-    for x in range(random.randint(a, b)):
-        size = c
+def create_list_of_dict(random_number_from, random_number_to, size_dictionary=4, random_letter_from = 5, random_letter_to = 1):
```

Comment:
This function is not stable. Why is default random_letter_to equal to 1?
Try this one to test:
create_list_of_dict(random_number_from = 2, random_number_to = 10)